script/check-style

```diff
@@ -33,14 +33,14 @@ if [[ ! -x "${CLANG_FORMAT}" ]]; then
     echo "Unsupported environment." ; exit 1 ;
   fi
 
-  echo "clang-bin: http://releases.llvm.org/${CLANG_VERSION_REQUIRED}/clang+llvm-${CLANG_VERSION_REQUIRED}-${CLANG_BIN}"
```

Comment:
The http URL redirects to the https URL now, which caused this to fail for me.

Reply:
Thanks for fixing this.